### PR TITLE
change Ankr to Mirage in different unity / unreal docs where needed

### PR DESCRIPTION
**Pull request by Antivortex**

### CODE REVIEW

File: pages/unreal-sdk/getting-started/before-you-start.md [hunk 1]
```diff
@@ -29,37 +29,37 @@ Currently, the SDK supports the following functionality for Unreal Engine:
 
 ## SDK Installation
 
-1. Download the `AnkrSDK.zip` package from the latest [release](https://github.com/Ankr-network/game-unreal-sdk/releases).
-2. Unzip the `AnkrSDK.zip` package to your Unreal Project's **Plugins** folder.
+1. Download the `MirageSDK.zip` package from the latest [release](https://github.com/Ankr-network/game-unreal-sdk/releases).
```
Comment (Vukiz) [inline pages/unreal-sdk/getting-started/before-you-start.md:32]: before we can update that we should change the unreal SDK first.

File: pages/extra/events-and-subscriptions.md [hunk 2]
```diff
@@ -143,8 +143,8 @@ Subscribing works via WebSockets only. Request a relevant endpoint from your pro
 First, instantiate a subscriber:
 
 ```
-var ankrSDK = AnkrSDKWrapper.GetSDKInstance("https://...");
-var _eventSubscriber = ankrSDK.CreateSubscriber("wss://...");
+var mirageSDK = MirageSDKWrapper.GetSDKInstance("https://...");
```
Comment (Vukiz) [inline pages/extra/events-and-subscriptions.md:146]: is this our latest flow? @Antivortex

**Review by Vukiz — CHANGES_REQUESTED** [review]

File: pages/extra/events-and-subscriptions.md (hunk 2 above)
Comment (Antivortex) [reply pages/extra/events-and-subscriptions.md:146]: This is one of the up-to-date options. There is a simpler way to get a single event with a special EventAwaiter object (i.e. a single log coming from a single smart contract method call) but this flow is valid too. EventAwaiter just wraps the EventSubscriber for a single event handling.

File: pages/unreal-sdk/getting-started/before-you-start.md (hunk 1 above)
Comment (Antivortex) [reply pages/unreal-sdk/getting-started/before-you-start.md:32]: It is already renamed to MirageSDK in the Unreal SDK repo latest releases, so this change makes it up to date

**Review by Vukiz — APPROVED** [review]

**Pull request merged**